hw2/01-coins.js

```diff
@@ -1,7 +1,84 @@
+/**
+ * Naya Mairena
+ * Fall 2022
+ * CS465P Full Stack
+ */
 /** Exercise 01 - Coins **/
 
+//Function that will use a conditional (ternary) operator to check if the change name needs to be plural.
+const plurals = (count, word) => {
+  //Check if last letter of the string ends with a "y". This is because "penny" is different than the others.
+  if (word.slice(-1) === 'y') {
+    //Slice "penny" to be "penn".
+    let newWord = word.slice(0, -1);
+    //Return "pennies" if there is more than 1 penny.
+    return count === 1 ? word : newWord + 'ies';
+  } else return count === 1 ? word : word + 's';
+  //Else, we want to return the word with an "s" at the end for multiples.
+};
+
+//Function that will take a dollar amount as input and return its change broken down by coins.
 const calculateChange = (input) => {
   // Add your code here
+  //Multiple use cases to check before actual functionality.
+  if (input > 10) return 'Error: the number is too large.';
+  else if (input === 0) return 'Error: 0 returns no change.';
+  else if (input < 0) return 'Error: Negative dollar amount not allowed.';
+  else if (isNaN(input)) return 'Error: Input is not a number value.';
+  else {
```

Comment:
Good on your for checking for all cases. I completely forgot! also, I like that you used isNaN because I think I typically will use typeof, but I think yours is more specific.